projekt-06-island-kikakoral-main/zadani/soft/style.css

```diff
@@ -0,0 +1,288 @@
+/* nejsme vcerejsi, pouzivame moderni box model */
+*, ::after, ::before { box-sizing: border-box; }
+
+/*
+ BARVY
+ pozadi           #f5f5f5
+ dekorace nadpisů #629edb
+*/
+
+
+body {
+    margin: 0;
+    padding: 0;
+    background-color: #f5f5f5;
+    font-family: 'Merriweather', serif;
+    font-size: 16px;
+    line-height: 1.6;
+}
+
+
+/* HLAVICKA */
+
+.hlavicka {
+    color: white;
+    background-image: url("/projekt-06-island-kikakoral-main/images/island-hlavicka.jpg");
+    background-repeat: no-repeat;
+    width:auto;
+    height: 400px;
+    background-position: center center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 0 ;
+    padding: 0 20vw
+}
+
+
+.titulek {
+    font-family: 'Raleway', serif;
+    font-size: 70px;
+    margin: 0;
+}
+
+.hr {
+    height: 2px;
+    background-color:white;
+    width: 60px;
+    opacity:0.5;
+}
+.perex {
+   font-size: 15px;
+   text-align: center;
+   padding: 15px;
+   text-shadow:dimgray;
+}
+
+.autor {
+    opacity:0.5;
+    font-style: italic;
+}
+
+
+/* MENU */
+
+.kontajner {
+    content: "";
+    display: block;
+    max-width: 960px;
+    background-color:#f5f5f5;
+    margin: -50px auto 0 auto;
+    position: relative;
+    /* padding: 20px; */
+}
+
+.boxik {
+    display: flex;
+    flex-direction: row;
+    padding: 20px;
+}
+
+.menu {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-content: stretch;
+    padding: 0 20px 0 20px;
+}
+
+.menu li {
+    padding: 8px;
+    text-transform:uppercase;
+    border-bottom: 1px solid grey;
+    font-family: 'Raleway', serif;
+    list-style-type: none; 
+}
+
+.menu li a {
+    text-decoration: none;
+    color: gray;
+    font-size: 13px;
+}
+
+nav {
+    min-width: fit-content;
+    border: 0;
+    padding: 15 px;
+}
+
+
+/* OBSAH */
+
+.uvod {
+    font-size: 13px;
+    text-shadow: 0 0 4px hsla(0, 0, 0, 0.2);
+}
+
+article img {
+max-width: 960px;
```

Comment:
Ten obrázek ti to na malém displayi slušně rozhazuje. Asi bych šáhla po max-width: 100%, ať se hezky přizpůsobí body a nevylýzá nám. :)